From 98145d814016dbc88705844aff3a036786807e4e Mon Sep 17 00:00:00 2001
From: Robin Gong <b38343@freescale.com>
Date: Wed, 17 Dec 2014 19:33:11 +0800
Subject: [PATCH] MLK-9891: ARM: dts: imx6sx-sabreauto: use WDOG_B pin to
 reset whole board

For the QSPI byte address not aligned in ROM code and kernel, we have to reset
power cycle to workaroud this issue. Use WDOG_B pin to trigger PWRON of pfuze.

Signed-off-by: Robin Gong <b38343@freescale.com>
(cherry picked from commit 86f82eadc612a746ab57760f78754e0619aa48b1)
(cherry picked from commit 9b85071287145bf16e8202c2be4fdf7a076f7f0d)
---
 arch/arm/boot/dts/imx6sx-sabreauto.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/imx6sx-sabreauto.dts b/arch/arm/boot/dts/imx6sx-sabreauto.dts
index 7661562..c8f7502 100644
--- a/arch/arm/boot/dts/imx6sx-sabreauto.dts
+++ b/arch/arm/boot/dts/imx6sx-sabreauto.dts
@@ -197,6 +197,7 @@
 				MX6SX_PAD_LCD1_DATA23__GPIO3_IO_24	0x80000000
 				MX6SX_PAD_SD2_CLK__GPIO6_IO_6		0x80000000
 				MX6SX_PAD_LCD1_DATA22__GPIO3_IO_23	0x80000000
+				MX6SX_PAD_GPIO1_IO13__WDOG1_WDOG_ANY	0x10b0
 			>;
 		};
 
-- 
1.8.0

